cmd/delete.go

```diff
@@ -41,9 +41,10 @@ var deleteCmd = &cobra.Command{
 	Use:   "delete [env-name] [-f <file-or-dir>]",
 	Short: "Delete Kubernetes resources described in local config",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) > 1 {
-			return fmt.Errorf("'delete' takes at most a single argument, that is the name of the environment")
+		if len(args) != 1 {
+			return fmt.Errorf("'delete' requires an environment name; use `env list` to see available environments")
 		}
+		env := args[0]
```

Comment:
There is a chance that args could be 0 since you only checking for != 1. This could panic.